meta-luneos/recipes-webos/rdxd/rdxd.bb

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2013-2019 LG Electronics, Inc.
+
+SUMMARY = "Remote diagnostics daemon and utilities"
+AUTHOR = "Gayathri Srinivasan <gayathri.srinivasan@lge.com>"
+SECTION = "base"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+DEPENDS = "glib-2.0 libpbnjson luna-prefs luna-service2 pmloglib"
+#Add tar dependency since --absolute-names support is missing in busybox tar
+RDEPENDS_${PN} = "nyx-utils tar"
+
+inherit webos_ports_fork_repo
+inherit webos_cmake
+inherit webos_system_bus
+inherit pkgconfig
+inherit webos_machine_impl_dep
+inherit webos_systemd
+
+WEBOS_GIT_PARAM_BRANCH = "webOS-ports/webOS-OSE"
+SRC_URI = "${WEBOS_PORTS_GIT_REPO_COMPLETE}"
+
+S = "${WORKDIR}/git"
+
+SRCREV = "f159cb54d13777e835015bdbcdf5be46e58fe539"
```

Comment:
set PV :)

Reply:
If I would know what to put before +git${SRCPV} that would help ;) I have no clue.... 1.0.0-3? 3.0.0-3?

Reply:
That comes from WEBOS_VERSION in the recipe you're importing (or version/X.Y.Z + submissions/N tags of the source version you've imported)